.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "modules/mavlink"]
+	path = modules/mavlink
+	url = https://github.com/mavlink/mavlink.git
```

Comment:
We are planning to keep a separate branch, mavlink-next, for messages that are not merged in master branch. Some messages related to video streaming were recently merged, so we're going to be able to support basic video streaming using upstream mavlink. In that branch I'll use a mavlink-fork